### DISCUSSION TITLE
logging using log4j

### DISCUSSION THREAD

**Discussion by jesserdhieb1** [NationalSecurityAgency/ghidra#5998] [Q&A] [answered]
Hello everyone,
I am new to ghidra plugin development, I wnated to just write a simple program that logs using log4j but I don't know how to use it with log4j, I always get an error , I used eclipce to develop the plugin.
thank you

**Comment by ryanmkurtz** [comment, marked as answer]
You should just be able to use the static methods in the `Msg` class to do your logging.

**Reply by jesserdhieb1** [reply]
thank you  !
